### ISSUE TITLE
runit.getParameter.R -> test.CheckParameterInSimulationNoneVariables not clear what it should do

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/R-Toolbox#30]
I cannot really understand what the above mentioned test should check for.

My first expectation whas that it checks whether certain parameters (of the type "current) can be accessed even when no parameters were initialized.

```
  dci_info <- standard_dci_info
  options <- list(Type="current")
  parameter <- getParameter(path_id=113, DCI_Info = dci_info, options=options)

  checkEquals(names(parameter), c("Value","Index"))
  checkEquals(dci_info$InputTab$AllParameters$Value[parameter$Index], parameter$Value)
```

According to the XML of the model, the parameter with path_id=113 is "black american girl|Organism|Muscle|Plasma|my Compound|concentration". When getting this parameter with its numerical ID (how performed in the test), everything works fine:

```
> parameter <- getParameter(path_id=113, DCI_Info = dci_info, options=options)
> parameter
$Value
[1] 0

$Index
[1] 2986
```

When trying to do the same with the path string (which is not performed in the tests, but should be!), the parameter cannot be found:

```
> parameter <- getParameter(path_id="black american girl|Organism|Muscle|Plasma|my Compound|concentration", DCI_Info = dci_info, options=options)
> parameter
$Value
numeric(0)

$Index
[1] 0
```

Problem is, 
```
  checkEquals(names(parameter), c("Value","Index"))
  checkEquals(dci_info$InputTab$AllParameters$Value[parameter$Index], parameter$Value)
```

still evaluate to TRUE, as `dci_info$InputTab$AllParameters$Value[0]` also returns numeric(0)! I.e., the test is always sucessfull.

So is there a bug in the test function, or am I getting the purpose of this test all wrong?

**Comment by PavelBal** [comment]
The unit test does not check for exceptions when trying to get a non-existent parameter. Have to add a respective check.

Also add tests for accessing parameters via path name.

**Comment by PavelBal** [comment]
Fixed with #34
